packages/fast-components-react-base/src/breadcrumb/breadcrumb.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { BreadcrumbClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import {
+    BreadcrumbHandledProps,
+    BreadcrumbProps,
+    BreadcrumbUnhandledProps,
+} from "./breadcrumb.props";
+import { get } from "lodash-es";
+
+class Breadcrumb extends Foundation<
+    BreadcrumbHandledProps,
+    BreadcrumbUnhandledProps,
+    {}
+> {
+    public static displayName: string = "Breadcrumb";
+
+    protected handledProps: HandledProps<BreadcrumbHandledProps> = {
+        children: void 0,
+        label: void 0,
+        separator: void 0,
+        managedClasses: void 0,
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): any {
+        return (
+            <nav
+                {...this.unhandledProps()}
+                aria-label={this.props.label}

```

Comment:
label is not required, I would suggest passing a null value if it doesn't exist.